.travis.yml

```diff
@@ -0,0 +1,93 @@
+# .travis.yml for use with EPICS Base ci-scripts
```

Comment:
The `.travis.yml` file has been removed from the 7.0 branch of Base, we've switched from Travis (which is no longer free) to using GitHub Actions instead, so please remove this file again.

Reply:
Removed